srcpkgs/reddio/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'reddio'
+pkgname=reddio
+version=0.3
+revision=5
+wrksrc="reddio-v0.3"
```

Comment:
use `wrksrc="reddio#v${version}"`, so you don't have to change it on every update